modules/machineset-creating.adoc

```diff
@@ -47,8 +47,6 @@ $ oc get machineset <machineset_name> -n \
 ....
     matchLabels:
       machine.openshift.io/cluster-api-cluster: agl030519-vplxk <1>
-      machine.openshift.io/cluster-api-machine-role: worker <2>
```

Comment:
<2> is still referred by `<2> A default node label`  bullet.

```
template:
    metadata:
      labels:
        machine.openshift.io/cluster-api-cluster: agl030519-vplxk <1>
        machine.openshift.io/cluster-api-machine-role: worker <2>
        machine.openshift.io/cluster-api-machine-type: worker
        machine.openshift.io/cluster-api-machineset: agl030519-vplxk-worker-us-east-1a
```

will do the trick.